torch/ao/quantization/backend_config/onednn.py

```diff
@@ -355,28 +353,27 @@ def _fuse_conv_bn_add_relu_left(is_qat, relu, add_pattern):
 
 
 def _conv_bn_add_relu_root_node_getter_left(pattern):
-    relu, add_pattern = pattern
+    _relu, add_pattern = pattern
     _, bn_conv, _ = add_pattern
-    bn, conv = bn_conv
+    _bn, conv = bn_conv
     return conv
 
 
 def _conv_bn_add_relu_extra_inputs_getter_left(pattern):
     """get inputs pattern for extra inputs, inputs for root node
     are assumed to be copied over from root node to the fused node
     """
-    relu, add_pattern = pattern
-    _, bn_conv, extra_input = add_pattern
-    bn, conv = bn_conv
+    _relu, add_pattern = pattern
+    _, _bn_conv, extra_input = add_pattern
     return [extra_input]
 
 
-conv_add_relu_left_optioins = itertools.product(
+conv_add_relu_left_options = itertools.product(
```

Comment:
This is technically changing a public API so would need to go through deprecation process.
Given that this code is most likely not maintained closely, I would just leave it as-is to avoid any issue.

Reply:
Urg, why did I do that?! (Changing the API) Compulsive desire to clean but out of scope! 🙂